fix: UTC-228: OOM prompts sending batches to Adala

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/17165088169)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/540
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17160472019

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17161525105

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-utc-228/pt-2'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17161525105

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17161620070

hakan458 commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17162359410)

hakan458 committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17162359410

hakan458 committed:
Merge branch 'develop' into 'fb-utc-228/pt-2'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17162359410

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17162539321

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17162966561

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17163320942

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17164498528

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-utc-228/pt-2'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17164498528

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17164539708

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17165088169
